chain/network/src/store/schema/mod.rs

```diff
@@ -20,7 +20,8 @@ pub struct AccountIdFormat;
 impl Format for AccountIdFormat {
     type T = AccountId;
     fn encode<W: io::Write>(a: &AccountId, w: &mut W) -> io::Result<()> {
-        w.write_all(a.as_ref().as_bytes())
+        let account_id_ref: &AccountIdRef = a.as_ref();
+        w.write_all(account_id_ref.as_bytes())
```

Comment:
Try this one:

```suggestion
        w.write_all(a.as_bytes())
```

If it works ([it should](https://github.com/near/near-account-id/blob/main/src/account_id_ref.rs#L76)), apply everywhere else